extension/background.js

```diff
@@ -0,0 +1,10 @@
+
+var optionValue = "aaa"
+
+chrome.runtime.onMessage.addListener(function(request, sender, sendResponse) {
```

Comment:
is this code given to you or did you write this?

what is `chrome.runtime.onMessage.addListener` all about?